recipe/activate.sh

```diff
@@ -6,34 +6,70 @@
 
 # doesn't come with a deactivate script, because the symlink
 # is benign and doesn't need to be deleted.
+CF_LIBARROW_ACTIVATE_LOGGING=1
+_la_log() {
+    if [ "$CF_LIBARROW_ACTIVATE_LOGGING" = "1" ]; then
+        # The following loop is necessary to handle multi-line strings
+        # like for the output of `ls -al`.
+        while IFS= read -r line; do
+            echo "${BASH_SOURCE[0]} DEBUG: $line"
+        done <<< "$*"
+    fi
+}
+
+_la_log "Beginning libarrow activation."
 
 # where the GDB wrappers get installed
-GDB_PREFIX="$CONDA_PREFIX/share/gdb/auto-load"
+_la_gdb_prefix="$CONDA_PREFIX/share/gdb/auto-load"
 
 # this needs to be in sync with ARROW_GDB_INSTALL_DIR in build.sh
-PLACEHOLDER="replace_this_section_with_absolute_slashed_path_to_CONDA_PREFIX"
+_la_placeholder="replace_this_section_with_absolute_slashed_path_to_CONDA_PREFIX"
 # the paths here are intentionally stacked, see #935, resp.
 # https://github.com/apache/arrow/blob/master/docs/source/cpp/gdb.rst#manual-loading
-WRAPPER_DIR="$GDB_PREFIX/$CONDA_PREFIX/lib"
+_la_symlink_dir="$_la_gdb_prefix/$CONDA_PREFIX/lib"
+_la_orig_install_dir="$_la_gdb_prefix/$_la_placeholder/lib"
 
-mkdir -p "$WRAPPER_DIR" || true
-# If the directory is not writable, nothing can be done
-if [ ! -w "$WRAPPER_DIR" ]; then
-    return
-fi
+_la_log "          _la_gdb_prefix: $_la_gdb_prefix"
+_la_log "         _la_placeholder: $_la_placeholder"
+_la_log "         _la_symlink_dir: $_la_symlink_dir"
+_la_log "    _la_orig_install_dir: $_la_orig_install_dir"
+_la_log "  content of that folder:"
+_la_log "$(ls -al "$_la_orig_install_dir" | sed 's/^/      /')"
 
-# there's only one lib in that folder, but the libname changes
+# there's only one lib in the _la_placeholder folder, but the libname changes
```

Comment:
Can be done after we remove/revert `CF_LIBARROW_ACTIVATE_LOGGING=1` again.
```suggestion
# there's only one lib in the _la_orig_install_dir folder, but the libname changes
```

Reply:
Got it. Updated the [todo list](https://github.com/conda-forge/arrow-cpp-feedstock/pull/1065#issuecomment-1568210670).

Reply:
Should we also rerender?

Reply:
No, this feedstock was very recently rerendered, and I don't like squashing your changes with those of the bot.